runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperConfig.java

```diff
@@ -136,6 +136,17 @@ default boolean swapLastDestination()
 
 	@ConfigItem(
 		position = 9,
+		keyName = "swapLastDestinationZanaris",
+		name = "Last-destination (XXX) on Fairy ring in Zanaris",
+		description = "Swap Configure with Last-destination on the Fairy ring in Zanaris"
+	)
+	default boolean swapLastDestinationZanaris()

```

Comment:
This should not be toggle, but should apply while having the other fairy ring option enabled

Reply:
As per my comment in #1639, I do believe we want to have this toggle. Imagine fishing for Karambwan while banking in Zanaris. In Zanaris you'd want to use Last-destination to go back to the Karambwan spot, but to go from the Karambwan spot to Zanaris you'd want to use the Zanaris option.

Reply:
That is too specific, and banking in zanaris while fishing Karambs is really not the best way to fish them.

Reply:
As an Ironman with limited access to jewelry, I beg to differ.